### ISSUE TITLE
NTR: Torsion of appendix of testis

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#6211]
**Preferred term label:**
Torsion of appendix of testis

**Synonyms**
hydatid of Morgagni (male), twisted vestigial remnant of the Müllerian duct

**Definition (free text, please give PubMed ID)**
The appendix testis or hydatid of Morgagni, is a vestigial remnant of the Müllerian duct located on the superior pole of the testicle between the testis and epididymis. Although this appendage has no vital function, it is present in 50% of males. Torsion of twisting of this vestigial remnant can be incredibly painful, but usually requires no medical intervention [PMID: 31550101](https://pubmed.ncbi.nlm.nih.gov/31550101/).

**Parent term (use hpo.jax.org/app)**
Abnormal testis morphology (`HP_0000035`)  

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Testicular Torsion ([`OMIM_187400`](https://www.omim.org/entry/187400?search=torsion%20of%20appendix%20of%20testis&highlight=%28testicular%7Ctestis%29%20appendix%20of%20torsion))  

**Your nano-attribution (ORCID)**  
https://orcid.org/0000-0002-8169-9049

**Comment by pnrobinson** [comment]
added term. Will add annotation after next release
